### DISCUSSION TITLE
Not receiving data

### DISCUSSION THREAD

**Discussion by villamanuel1980** [revk/xESP32-Faikout#144] [Q&A]
We connected our Faikin to an indoor VRV unit.

First the faikin seemed not to get the correct voltage, and blinked repeteadly but didn't go into the wifi.

Then we fed it 12v from an external transformer, and it started working. The other three wires are connected to the indoor unit.

Faikin is working but communication doesn't seem to work. Funnily enough, the other thermostat now shows an U8 error, so it knows something has changed and there's now a second remote connected to "wireless".

We're wondering if we need to change the settings in the indoor unit or the outdoor unit to indicate that a wireless remote is now present? and how will it decide which remote takes precedence?

It feels like we're close, but not quite there. Can someone help?

Thanks!!

**Comment by revk** [comment]
OK is this an S21 connector - they normally have the 12V (ish) power needed. What model is it (someone else may have used that model and have ideas). etc.

**Comment by villamanuel1980** [comment]
It says "wireless" on the indoor unit, not s21. It does have 5 pins and
look like a s21 and when connected to the faikin the faikin led blinked
continuosly but didn't go into the WiFi so we believe it didn't have enough
power and gave it 12v from the outside.

We connected it with Dupont cables, and it's now unclear where each of the
other three pins from Faikin have to go in the indoor unit.

The indoor unit is FXSQ32M8V3B .

I feel we're missing something basic here. Communication should work with
just two pins connected right? Are these always the same two in the indoor
unit or do they change? Is this daikin trying to make it hard for us or is
this something that I'm missing here?

Thanks!!

**Reply by revk** [reply]
I don't know. The S21 has a specific pin out. The faikin needs GND, power, and tx and rx to work. But only works if the unit talks the right serial protocol which is in S21 or X50A.

Over to anyone else that knows that model, sorry.

**Reply by villamanuel1980** [reply]
The thing is that the schematic doesn't say S21 anywhere, and there's no X50A either. (there's X5A).

The closest we have is that port named "Wireless".

**Comment by villamanuel1980** [comment]
What is the worst case scenario if we connect a tx and rx from the faikin
to the wrong s21 pin? (the one with 12v, I guess?).

Will this burn the Faikin?

I guess my question is: can we brute force this and try every pin
combination? or will we destroy the Faikin in the process? is there any way
to find out which pins are tx and rx in the board if it doesn't clearly say
so?

Thanks!!!

**Reply by dzungpv** [reply]
You should upload picture of the indoor unit here, so we can check it, now the board support only S21 and X50A port

**Reply by revk** [reply]
And you can check voltages with a meter first!

**Comment by villamanuel1980** [comment]
Yeah I will check with the multimeter on the indoor unit.

I don't know what the ground pin is, so I can measure with black in
anything metallic and red for each pin, right?

What should the measures be so I know which is which on the s21?

Grd 0
Line 12v
Neutral?
Tx?
Rx?

And this go to Faikin either in the same or I'm reverse order, right?


As you can see, I'm way over my hear here. Love tinkering, but I'm missing
some basic stuff.. XD

**Reply by revk** [reply]
GND is pin 5.
Pin 1 is usually 5V
Pin 4 is usually more like 12V or more
Pins 2 and 3 are likely to be around 5V

**Comment by villamanuel1980** [comment]
![WhatsApp Image 2023-11-09 at 07 51 27 (2)](https://github.com/revk/ESP32-Faikin/assets/69581102/1b170590-e489-4e5d-953c-7461e42b7cea)

So this is the port we're trying to connect to. It says "Wireless" and seems to have the 5 pins that correspond to an s21 port.

We couldn't find any others that may seem like it.

This is the schematic in the indoor unit.

![WhatsApp Image 2023-11-09 at 07 51 27 (1)](https://github.com/revk/ESP32-Faikin/assets/69581102/4366016c-e440-4924-b7b7-e391da2a3d95)

This is the port from a little further.

![WhatsApp Image 2023-11-09 at 07 51 23](https://github.com/revk/ESP32-Faikin/assets/69581102/4d1142d1-dd73-4f48-b97e-6fcee9afb172)

And this is the Faikin with the Duponts. We tried to connect it like the picture here and the Faikin blinked white all the time. It stopped when we put an external 12v on it and then the Faikin worked but said "no communication", while the Wall Thermostat gave a new error that another controller was present.

I believe the blinking white could be that the wiring was fortituosly giving it 5v that was enough to try to boot up but not to actually do? I'm not certain on how to connect the three pins that are not bringing current to the Faikin, that's where we're at. Next time I can go to the unit, I'll measure the pins and let you know what we accomplish!!

Thank you all! This is a lot of fun but also kind of stressful, uh? XD
